scripts/setup.sh

```diff
@@ -179,19 +205,58 @@ get_ext_files()
   done
 }
 
+get_extensions_zip_packages() {
+  curl -s -O $EXTENSION_S3_URL/$EXTENSION_MANIFEST_FILE
```

Comment:
where is this variable set?

Reply:
https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/175/files#diff-ce435f38623d032d25670f2c63a430dfd1a02db661d10c1ef031cbef2f0f0ef0R81